test/unit/mocked_data/test_get_bgp_neighbors/normal/expected_result.json

```diff
@@ -4,7 +4,7 @@
     "peers": {
       "192.168.56.4": {
         "is_enabled": true,
-        "uptime": 1452696202,
```

Comment:
No, please do not change the tests under the ```normal``` test case.
As this is a corner case, not the default behaviour, please create a new test case for the output you see.